website/static/js/projectSettings.js

```diff
@@ -67,19 +101,45 @@ var NodeCategorySettings = oop.extend(
                 .then(function(response) {
                     return response.updated_fields.category;
                 })
-                .done(self.updateSuccess.bind(self))
-                .fail(self.updateError.bind(self));
+                .done(self.updateCategorySuccess.bind(self))
+                .fail(self.updateCategoryError.bind(self));
+        },        
+        updateTitle: function() {
+            var self = this;
+            return $osf.putJSON(self.titleDescriptionEditUrl, {
+                    title: $osf.htmlEscape(self.title()),
+                })
+                .done(self.updateDescription.bind(self))
```

Comment:
Right - what I'm not understanding is why you're calling `self.updateDescription` inside `updateTitle`. Either the wrong function is getting called, or the success funciton should be renamed to show that it gets called after updating more than just the description.